lib/trollop.rb

```diff
@@ -37,6 +37,203 @@ class VersionNeeded < StandardError
 ## Regex for parameters
 PARAM_RE = /^-(-|\.$|[^\d\.])/
 
+class Option
+
+  attr_accessor :name, :short, :long, :default
+  attr_writer :multi_given
+  
+  def initialize
```

Comment:
what about passing in `educate` and `multi?` from the child classes?

Does that cut down on the number of lines / simplify this?